.gitignore

```diff
@@ -68,3 +68,7 @@ google-services.json
 
 # Android Profiling
 *.hprof
+
+
+arduino/libraries/
+arduino/cmake-build-debug/
```

Comment:
Maybe add these to #90 instead?

Reply:
I'm not sure how I'm supposed to add it. Should I close this pull request and push it again to pull request #90?

Reply:
Checkout the branch that #90 is on, make the same changes, commit and push them